src/Common/src/TypeSystem/Common/SignatureVariable.cs

```diff
@@ -64,7 +64,7 @@ public override int GetHashCode()
 
         protected override TypeFlags ComputeTypeFlags(TypeFlags mask)
         {
-            TypeFlags flags = 0;
+            TypeFlags flags = TypeFlags.ContainsGenericVariablesComputed | TypeFlags.ContainsGenericVariables;
```

Comment:
Only GenericParameterDesc is a generic variable. This should not be needed. We should not need to ask this question.